pkg/utils/utils.go

```diff
@@ -267,3 +267,28 @@ func FindFreePort() (int, error) {
 
 	return port, nil
 }
+
+func DeleteOldPackages(pkgPath, pkgType string) error {
```

Comment:
Please add detailed comments for this function.

Reply:
Done